spec/02-integration/04-admin_api/10-services_routes_spec.lua

```diff
@@ -191,6 +191,14 @@ for _, strategy in helpers.each_strategy() do
               pages[i] = json
             end
           end)
+          it("propagate in next a page size", function()
```

Comment:
style: add a blank line before `it(...)`

Reply:
Fixed :smile_cat: